alune/screen.py

```diff
@@ -48,6 +49,52 @@ def get_button_on_screen(
     return get_on_screen(image, button.image_path, button.capture_area, precision)
 
 
+def get_image_from_path(path: str) -> MatLike | None:
+    """
+    Get an image at a path.
+
+    Args:
+        path: The relative or absolute path to the image to be found.
+
+    Returns:
+        The image or none if it does not exist.
+    """
+    image_to_find = cv2.imread(path, 0)
+
+    if image_to_find is None:
+        logger.warning(f"The image {path} does not exist on the system " f"or we do not have permission to read it.")
```

Comment:
Why are we splitting the string up (via `" f"`)? 🤔

Reply:
Not intentional, my brain has a little different opinion than black auto format sometimes